### PR TITLE
SC-588 compiler get functions doc throws error

### PR DESCRIPTION
**Pull request by mrkraft**

### CODE REVIEW

File: lang/doc/v4/funcs/encoding-and-decoding-functions.hjson [hunk 1]
```diff
@@ -6,24 +6,186 @@
...
+      paramsDoc: [ "The array of bytes to encode." ]
+    }
     {
       name: "groth16Verify"
       params: [ "ByteVector", "ByteVector", "ByteVector" ]
       doc: "[Zero-knowledge proof](https://en.wikipedia.org/wiki/Zero-knowledge_proof) verifier function."
       paramsDoc: [ "Verifying key.", "Proofs.", "Inputs." ]
     }
+    {
+      name: "groth16Verify_1inputs"
```
Comment (msmolyakov) [inline lang/doc/v4/funcs/encoding-and-decoding-functions.hjson:100]: maybe `grothVerify` should be in `verification-functions.hjson`?

**Comment by msmolyakov** [comment]
Missed `median` function

File: lang/doc/src/test/scala/com/wavesplatform/utils/DocExportTest.scala [hunk 2]
```diff
@@ -9,18 +9,33 @@ import com.wavesplatform.lang.directives.values.{Account, Expression, StdLibVers
...
 import com.wavesplatform.utils.doc.RideFullContext
-import org.scalatest.{Matchers, PropSpec}
+import org.scalatest.{FreeSpec, Matchers, PropSpec}
 import org.scalatestplus.scalacheck.{ScalaCheckPropertyChecks => PropertyChecks}
 
 class DocExportTest extends PropSpec with PropertyChecks with Matchers {
-  property("declared ride funcs and vars have doc for all contexts") {
-    val totalDocs = for {
+
+  property("declared ride FUNCs have doc for all contexts") {
```
Comment (msmolyakov) [inline lang/doc/src/test/scala/com/wavesplatform/utils/DocExportTest.scala:17]: Please add `DocExportTest` to unit tests scope

**Comment by msmolyakov** [comment]
Also please fix descriptions of getBinary/getBoolean/getInteger/getString functions in `extracting-data-functions.hjson`: remove `"and extract"` part, because they don't extract values and return `T|Unit`

**Comment by mrkraft** [comment]
> Also please fix descriptions of getBinary/getBoolean/getInteger/getString functions in `extracting-data-functions.hjson`: remove `"and extract"` part, because they don't extract values and return `T|Unit`
Fixed

**Pull request merged**